armi/reactor/blueprints/tests/test_blockBlueprints.py

```diff
@@ -297,6 +311,15 @@ def test_blockLattice(self):
                 seen = True
         self.assertTrue(seen)
 
+    def test_componentsNotInLattice(self):
+        """
+        Ensure that we catch cases when a component is supposed to be in the grid,
+        but is not.
+        """
+        with io.StringIO(FULL_BP_WRONG_LATTICE_MAP) as stream:
+            self.blueprints = blueprints.Blueprints.load(stream)
+            self.blueprints._prepConstruction(self.cs)
```

Comment:
Sorry, are you saying this should fail, but it doesn't?

Perhaps I am confused. Maybe adding an `assert` to this test would make it more clear. That is one of our testing standards, anyway:

https://github.com/terrapower/armi/blob/cf9106d9142a8dda298e6e0c73dc0875233179bf/doc/developer/standards_and_practices.rst?plain=1#L220

Reply:
Please ignore this, I've updated the PR and that test has been updated to actually _test_ something.